libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -72,6 +72,21 @@ class elasticsearch_plugin_impl
       std::string bulk_line;
       std::string index_name;
       bool is_sync = false;
+      const double lookup_pow_table[13] = {
+              1,
+              10,
+              100,
+              1000,
+              10000,
+              100000,
+              1000000,
+              10000000,
+              100000000,
+              1000000000,
+              10000000000,
+              100000000000,
+              1000000000000
```

Comment:
Hm, not sure if the last three require LL modifier?

Reply:
FYI there is already a lookup table exists in asset.hpp. No need to reinvent the wheel.

Reply:
nice one @abitmore , using that now :)